spec/pQueue.js

```diff
@@ -0,0 +1,122 @@
+import chai, { expect } from 'chai'
```

Comment:
Because I am very ignorant of the testing, I could only review you based on the knowledge I know so sorry about all this.